bookshop/srv/cat-service.cds

```diff
@@ -6,5 +6,5 @@ service CatalogService @(path:'/browse') {
   } excluding { createdBy, modifiedBy };
 
   @requires_: 'authenticated-user'
-  action submitOrder (book : Books.ID, amount: Integer);
+  action submitOrder (book : type of Books:ID, amount: Integer);
```

Comment:
So why not go for just `Books:ID`?

Reply:
Hi Christian,

the VSCode Plugin marked it as an error so I assumed that it uses an older cds-compiler version. But I had a look at their changelog and it states that cds-compiler 1.42.2 is used... I have no idea why it still displays it as an error.

But I would go with the safe version just in case.


Regards,
Andre

Reply:
Could not reproduce the error w/ latest CDS plugin from today (using compiler 1.45.0).  Can you verify again?

Reply:
Can confirm. The VS Code plugin in version v3.1.0 works with `book: Books:ID` 😃